### PR TITLE
PD-1634 / 24.10 / Pd 1634 24 10 1 docs

### PR DESCRIPTION
**Pull request by DjP-iX**
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

**Comment by DjP-iX** [comment]
backport

File: content/GettingStarted/SCALEReleaseNotes.md [hunk 1]
```diff
@@ -154,8 +154,38 @@ Any new feature flags introduced since the previous OpenZFS version that was int
...
 For more details on feature flags, see [OpenZFS Feature Flags](https://openzfs.github.io/openzfs-docs/Basic%20Concepts/Feature%20Flags.html) and [OpenZFS zpool-feature.7](https://openzfs.github.io/openzfs-docs/man/7/zpool-features.7.html).
 
+## 24.10.1 Changelog
+
+**December 12, 2024**
+
+iXsystems is pleased to release TrueNAS 24.10.1!
+This is a maintenance release and includes refinement and fixes for issues discovered after the 24.10.0 and 24.10.0.X releases.
+
+* Prevent incorrect translation of LDAP basedn to kerberos realm ([NAS-132192](https://ixsystems.atlassian.net/browse/NAS-132192)).
```
Comment (chrismillsapsix) [inline content/GettingStarted/SCALEReleaseNotes.md:164]: Assuming "basedn" was taken from the referenced ticket, but we use "Base DN" in other parts of the docs. 
https://www.truenas.com/docs/scale/24.10/scaleuireference/credentials/directoryservices/ldap/

File: content/GettingStarted/SCALEReleaseNotes.md [hunk 2]
```diff
@@ -154,8 +154,38 @@ Any new feature flags introduced since the previous OpenZFS version that was int
...
+* Increase the maximum permitted Samba (SMB) ACL size from 64 to 1024 entries ([NAS-132344](https://ixsystems.atlassian.net/browse/NAS-132344)).
+* Fixed bug: Applications service fails after upgrade if app requires GPU, seemingly due to missing Nvidia Drivers ([NAS-132070](https://ixsystems.atlassian.net/browse/NAS-132070) and [NAS-132131](https://ixsystems.atlassian.net/browse/NAS-132131)).
```
Comment (linzi-ix) [inline content/GettingStarted/SCALEReleaseNotes.md:166]: This is nit-picky, but all of these begin with a present-tense verb apart from "Fixed." Might make the flow feel smoother if we do the same for that one.

File: content/GettingStarted/SCALEReleaseNotes.md [hunk 3]
```diff
@@ -153,9 +151,58 @@ Any new feature flags introduced since the previous OpenZFS version that was int
 {{< /truetable >}}
 
 For more details on feature flags, see [OpenZFS Feature Flags](https://openzfs.github.io/openzfs-docs/Basic%20Concepts/Feature%20Flags.html) and [OpenZFS zpool-feature.7](https://openzfs.github.io/openzfs-docs/man/7/zpool-features.7.html).
+<!--## 24.10.2 Changelog
+
+Remove midclt call from 24.10.1 known issues if [NAS-132914](https://ixsystems.atlassian.net/browse/NAS-132914) is complete -->
+
+## 24.10.1 Changelog
+
+**December 12, 2024**
```
Comment (TheJulianJES) [inline content/GettingStarted/SCALEReleaseNotes.md:160]: Should this date be changed to the new release date?
```suggestion
**December 17, 2024**
```

**Pull request merged**